recommenders/README.md

```diff
@@ -39,25 +39,25 @@ Note that, currently, NNI and Vowpal Wabbit are in the experimental group.
 These groups can be installed alone or in combination:
 ```bash
 # install recommenders with core requirements and support for CPU-based recommender algorithms and notebooks
-pip install recommenders[examples]
+pip install --no-cache --no-binary scikit-surprise recommenders[examples]
 
 # add support for running example notebooks and GPU functionality
-pip install recommenders[examples,gpu]
+pip install --no-cache --no-binary scikit-surprise recommenders[examples,gpu]
 ```
 
 ## GPU Support
 
-You will need CUDA Toolkit v10.0 and CuDNN >= 7.6 to enable both Tensorflow and PyTorch to use the GPU. For example, if you are using a conda enviroment, this can be installed with
+You will need CUDA Toolkit v11.2 and CuDNN v8.1 to enable both Tensorflow and PyTorch to use the GPU. For example, if you are using a conda enviroment, this can be installed with
 ```bash
-conda install cudatoolkit=10.0 "cudnn>=7.6"
+conda install -c conda-forge cudatoolkit=11.2 cudnn=8.1
 ```
 For a virtual environment, you may use a [docker container by Nvidia](../SETUP.md#using-a-virtual-environment). 
 
 For manual installation of the necessary requirements see [TensorFlow](https://www.tensorflow.org/install/gpu#software_requirements) and [PyTorch](https://pytorch.org/get-started/locally/) installation pages.
 
 When installing with GPU support you will need to point to the PyTorch index to ensure you are downloading a version of PyTorch compiled with CUDA support. This can be done using the --find-links or -f option below.
 
-`pip install recommenders[gpu] -f https://download.pytorch.org/whl/cu100/torch_stable.html`
+`pip install --no-cache --no-binary scikit-surprise recommenders[gpu] -f https://download.pytorch.org/whl/cu100/torch_stable.html`
```

Comment:
is this the correct location to load the torch 11.2 libraries? or should it be https://download.pytorch.org/whl/cu111/torch_stable.html
i couldn't find a 11.2 specific link from their [documentation](https://pytorch.org/get-started/locally/)

Reply:
Thanks, good catch. Tests are passing with either.
I substituted the later one where appropriate.
It results in installing torch 1.10.0+cu111.
It is strange, there is no 112 but there is 110, 111, 113 🤷